cmd/agent.go

```diff
@@ -64,5 +64,12 @@ func init() {
 		false,
 		"Runs agent a single time if true, or continously if false",
 	)
+	agentCmd.PersistentFlags().StringVarP(
+		&agent.DataFilePath,
+		"write-to-file",
```

Comment:
Do you think `--output-path` might be more descriptive?

Reply:
Yes, that's a better name